src/managers/channels.ts

```diff
@@ -66,4 +76,93 @@ export class ChannelsManager extends BaseManager<ChannelPayload, Channel> {
         .catch((e) => reject(e))
     })
   }
+
+  async sendMessage(
+    channel: string | TextChannel,
+    content?: string | AllMessageOptions,
+    option?: AllMessageOptions,
+    reply?: Message
+  ): Promise<Message> {
+    const channelID = typeof channel === 'string' ? channel : channel.id
+
+    if (typeof content === 'object') {
+      option = content
+      content = undefined
+    }
+    if (content === undefined && option === undefined) {
+      throw new Error('Either text or option is necessary.')
+    }
+    if (option instanceof Embed) {
+      option = {
+        embed: option
+      }
+    }
+
+    const payload: any = {
+      content: content,
+      embed: option?.embed,
+      file: option?.file,
+      files: option?.files,
+      tts: option?.tts,
+      allowed_mentions: option?.allowedMentions
+    }
+
+    if (reply !== undefined) {
+      const reference: MessageReference = {
+        message_id: reply.id,
+        channel_id: reply.channel.id,
+        guild_id: reply.guild?.id
+      }
+      payload.message_reference = reference
+    }
+
+    const resp = await this.client.rest.api.channels[channelID].messages.post(
+      payload
+    )
+    const chan =
+      typeof channel === 'string'
+        ? // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          (await this.get<TextChannel>(channel))!
+        : channel
+    const res = new Message(this.client, resp, chan, this.client.user as any)
+    await res.mentions.fromPayload(resp)
+    return res
+  }
+
+  async editMessage(
+    channel: string | TextChannel,
+    message: Message | string,
+    text?: string,
```

Comment:
make users to allow to use message options via text arg

Reply:
done